src/main/java/uk/gov/companieshouse/missingimagedelivery/orders/api/service/MissingImageDeliveryCostCalculatorService.java

```diff
@@ -27,17 +25,28 @@ public class MissingImageDeliveryCostCalculatorService {
 
     /**
      * Calculates the missing image delivery item costs.
-     * @param quantity the number of items
-     * @param productType product type based on category
+     *
+     * @param quantity                 the number of items
+     * @param productType              product type based on category
+     * @param userGetsFreeCertificates if user has permission admin/free-mids, return true
      * @return all of the relevant costs
      */
-    public ItemCostCalculation calculateCosts(final int quantity, final ProductType productType) {
+    public ItemCostCalculation calculateCosts(final int quantity, final ProductType productType, final boolean userGetsFreeCertificates) {
         checkArguments(quantity);
-        final int calculatedCost = costs.getMissingImageDeliveryItemCost();
-        final String totalItemCost = Integer.toString(quantity * calculatedCost);
+        final int basicCost = costs.getMissingImageDeliveryItemCost();
+        final int calculatedCost;
+        if (userGetsFreeCertificates) {
```

Comment:
This part of the code could be simplified and made clearer, something like:

`final var totalItemCost = userGetsFreeCertificates ? "0" : Integer.toString(quantity * calculatedCost);`

This way, you no longer need separate variables for discountApplied and itemCost, and you directly handle the final cost based on the user's permission.